### PR TITLE
Feature/us7410 authentication library

### PR DESCRIPTION
**Pull request by Ganonside**

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
```
Comment (silbermm) [inline .eslintrc.json]: do we still need this since we are using TS now?

File: spec/.eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
```
Comment (silbermm) [inline spec/.eslintrc.json]: same as last question... do we still need this?

File: .eslintrc.json (hunk 1 above)
Comment (Ganonside) [inline .eslintrc.json]: the test file is written in javascript so I left it in.

File: spec/.eslintrc.json (hunk 2 above)
Comment (Ganonside) [inline spec/.eslintrc.json]: using it for the tests

**Pull request merged**